defaults/main.yml

```diff
@@ -205,7 +205,7 @@ bbb_ufw_allow_networks: "{{ bbb_ufw_allow_networks_custom | default([]) + bbb_uf
 bbb_ufw_reject_networks: "{{ bbb_ufw_reject_networks_custom | default([]) + bbb_ufw_reject_networks_default }}"
 bbb_firewall_ufw:
   # enable/disable the ufw
-  enabled: true
+  enabled: false
```

Comment:
The testrun should not change this default.

Reply:
Good catch

Reply:
as said above, it has been creating issue on my side (dns/system-resolved).
unclear why because from my reading, there are no outbound firewall rules but.

reverted

Reply:
Following error when ufw enabled
https://github.com/juju4/ansible-role-bigbluebutton/runs/2823828866?check_suite_focus=true#step:6:3378
```
Failed to download key at https://ubuntu.bigbluebutton.org/repo/bigbluebutton.asc: Request failed: <urlopen error [Errno -3] Temporary failure in name resolution>
```

after failure info
https://github.com/juju4/ansible-role-bigbluebutton/runs/2823828866?check_suite_focus=true#step:9:241